src/Card/Card.js

```diff
@@ -13,7 +16,27 @@ const Card = ({ movieId, title, poster, rating, userRating, isLoggedIn }) => {
           alt="Movie Poster"
         />
       </Link>
-      <h3>{title}</h3>
+			<div>
+				{isLoggedIn && !isFavorite &&
+				<input
+					type='image'
+					alt='Star Icon'
+					src={whiteStar}
+					className='favorite'
+					onClick={(event) => changeFavoriteStatus(movieId, event)}>
+				</input>
+				}
+				{isLoggedIn && isFavorite &&
```

Comment:
Nice conditional rendering here! this lil feature is way trickier than I thought.